src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -2183,6 +2183,39 @@ template <typename GcInfoEncoding> void TGcInfoDecoder<GcInfoEncoding>::ReportRe
 
 #endif // Unknown platform
 
+#ifdef FEATURE_INTERPRETER
+template <> OBJECTREF* TGcInfoDecoder<InterpreterGcInfoEncoding>::GetStackSlot(
+                        INT32           spOffset,
+                        GcStackSlotBase spBase,
+                        PREGDISPLAY     pRD
+                        )
+{
+    OBJECTREF* pObjRef = NULL;
+
+    if( GC_SP_REL == spBase )
+    {
+        _ASSERTE(!"GC_SP_REL is invalid for interpreter frames");
+    }
+    else if( GC_CALLER_SP_REL == spBase )
+    {
+        _ASSERTE(!"GC_CALLER_SP_REL is invalid for interpreter frames");
+    }
+    else
+    {
+        _ASSERTE( GC_FRAMEREG_REL == spBase );
+
+        // HACK: The register slot we stash the frame pointer into varies per-architecture.
```

Comment:
I'm not sure I understand this comment and what's hacky about what we do here. I also don't know why we would want to use CONTEXTGetFp, the GetFP is the way that runtime uses for accessing frame pointer register in a generic way.

Reply:
The code was hacky previously, I forgot to remove the comment when I migrated it to GetFP. Thanks.